src/Uno.UWP/UI/Color.wasm.cs

```diff
@@ -17,14 +19,20 @@ partial struct Color
 
 		internal string ToHexString()
 		{
-			var result = "#";
+			var builder = new StringBuilder(10);
+
+			builder.Append("#");
+			builder.Append(R.ToString("X2"));
+			builder.Append(G.ToString("X2"));
+			builder.Append(B.ToString("X2"));
+
 			if (A != 255)
```

Comment:
You should use the condition to create the right `StringBuilder`'s capacity.

Reply:
After doing some perf testing, it is not significant.